src/js/common/schemaform/review/ReviewPage.jsx

```diff
@@ -42,6 +44,22 @@ class ReviewPage extends React.Component {
     }
   }
 
+  /*
+   * Returns the page list without conditional pages that have not satisfied
+   * their dependencies and therefore should be skipped.
+   */
+  getEligiblePages() {
+    const { form, route: { pageList, path } } = this.props;
+    const eligiblePageList = getActivePages(pageList, form);
+    const pageIndex = _.findIndex(item => item.pageKey === path, eligiblePageList);
+    return { eligiblePageList, pageIndex };
+  }
+
+  goBack() {
```

Comment:
Can we add a test for this method? I think there's one in the FormPage tests that should be similar.